src/Isotope80/Isotope.cs

```diff
@@ -562,6 +562,74 @@ static Unit disposeWebDriver(IWebDriver d)
             from _ in trya(() => d.Manage().Window.FullScreen(), "Failed to change browser to fullscreen")
             select unit;
 
+        /// <summary>
+        /// Navigate back using the browser's back button
+        /// </summary>
+        public static Isotope<Unit> back =>
+            from d in webDriver
+            from _ in trya(() => d.Navigate().Back(), "Failed to go back in browser")
+            select unit;
+
+        /// <summary>
+        /// Navigate forward using the browser's forward button
+        /// </summary>
+        public static Isotope<Unit> forward =>
+            from d in webDriver
+            from _ in trya(() => d.Navigate().Forward(), "Failed to go forward in browser")
+            select unit;
+
+        /// <summary>
+        /// Refresh current page
+        /// </summary>
+        public static Isotope<Unit> refresh =>
+            from d in webDriver
+            from _ in trya(() => d.Navigate().Refresh(), "Failed to refresh current page")
+            select unit;
+
+        /// <summary>
+        /// Opens and switches to new tab
+        /// </summary>
+        public static Isotope<Unit> newTab =>
+            from d in webDriver
+            from _ in trya(() => d.SwitchTo().NewWindow(WindowType.Tab), "Failed to open new tab")
+            select unit;
+
+        /// <summary>
+        /// Opens and switches to new tab, then executes Isotope
+        /// </summary>
+        public static Isotope<T> NewTab<T>(this Isotope<T> iso) =>
```

Comment:
These `NewTab` and `NewWindow` extension methods kind of change the pattern of use of Isotope which I think needs a bit more thought. Can we take them out for now please.